TypeScript/test/items.mock.ts

```diff
@@ -0,0 +1,34 @@
+import { Item } from "../app/rose-garden";
+
+export const items = [
+  new Item("+5 Dexterity Vest", 10, 20),
+  new Item("Aged Brie", 2, 0),
+  new Item("Elixir of the Mongoose", 5, 7),
+  new Item("Sulfuras, Hand of Ragnaros", 0, 80),
+  new Item("Sulfuras, Hand of Ragnaros", -1, 80),
+  new Item("Backstage passes to a TAFKAL80ETC concert", 15, 20),
+  new Item("Backstage passes to a TAFKAL80ETC concert", 10, 49),
+  new Item("Backstage passes to a TAFKAL80ETC concert", 5, 49),
+  new Item("Example not tested", 15, 21),
+];
+
+const sanitizeExpectedMockTest = (item: Item) =>
```

Comment:
I was thinking that this function is not totally necessary, since what we need is to return the `Item` instance and not the object with the parameters as I currently do... But in case we want to return an object with the parameters, for example to send them to a database, I think it could be a good implementation.

But assuming we need to return the `Item` instance then we should change the `return` of the `.map()` of the `updateQuality` function.
An example could be the following:
```js
if (item.sellIn < 0) {
     item.quality = this.sanitizeQuality(item.quality - 2)
     return item; // Maybe this return must be at the end of the .map() to make it once
}
```